orsopy/fileio/base.py

```diff
@@ -431,6 +431,9 @@ def represent_data(self, data):
         elif isinstance(data, datetime.datetime):
             value = data.isoformat("T")
             return super().represent_scalar("tag:yaml.org,2002:timestamp", value)
+        elif (not np.shape(data)) and hasattr(data, "item"):
```

Comment:
how about `elif (np.isscalar(data) and hasattr(data, "item"):`?

Reply:
done.